.eslintrc.js

```diff
@@ -1,9 +1,10 @@
 module.exports = {
   root: true,
+  parser: '@typescript-eslint/parser',
   parserOptions: {
-    ecmaVersion: 2017,
+    ecmaVersion: 2020,
```

Comment:
Should we keep this as `2017` and then add a `2020` in the `**/*.ts` override?

Reply:
Yeah, sounds good!